### PR TITLE
feat(sequelize): add health indicator

### PR DESCRIPTION
**Pull request by Tony133**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information
This PR added Sequelize Health Indicator

### CODE REVIEW

**Comment by BrunnerLivio** [comment]
Hey @Tony133, looks really promising so far. Could you do me a favor and rebase with the latest changes from master? I've totally forgotten to run tests during pull requests. Just want to make sure your changes run in the CI env :)

**Comment by Tony133** [comment]
Hi @BrunnerLivio,  then I move all the changes and I pull you on the master branch?

**Comment by Tony133** [comment]
Hi @BrunnerLivio  I did as you asked 

this pull request #901 

Tony

**Comment by BrunnerLivio** [comment]
@Tony133 alright lets close this one then.

For next time; you can easily sync changes from master with keeping the same PR open, like this:

```bash
# Since you are working on a fork, you need to add the parent repository as remote
# Only needs to be done once in your git fork
git remote add upstream https://github.com/nestjs/terminus.git

# Get the latest changes (but don't apply them)
git fetch --all

# This applies the latest changes from the parent repo to your current branch, without creating a merge commit
# Make sure everything is committed, otherwise, rebase will complain
git rebase upstream/master

# Update the latest changes
# You need to use --force since you "rewrote" history. It's fine to use --force, since you are working on a fork.
git push origin master --force
```

**Comment by Tony133** [comment]
Hi @BrunnerLivio, I always do it only this time I forgot to do it, my fault. :(

I resolved the conflicts

**Comment by Tony133** [comment]
I also resolved the conflicts on the master

P.s. I wanted to ask you, if you can add me to the repo as a contributor? always if you can

**Comment by Tony133** [comment]
Do I run the conflicts, @BrunnerLivio if we used this branch instead of the master branch?

**Comment by BrunnerLivio** [comment]
> P.s. I wanted to ask you, if you can add me to the repo as a contributor? always if you can

Let's exchange on Discord: `BrunnerLivio#8837`

I'll try to review your PRs asap, thanks a lot!!

**Comment by Tony133** [comment]
I am also on Discord (```Tony133#1430```), in the meantime I have done other pulls on terminus and some in other NestjJS packages :-)

Write to me as soon as you can no problem :-)
I'm also on Dev.to I'm publishing some posts on NestJS, and I have others I'm preparing --> https://dev.to/tony133

File: package.json [hunk 1]
```diff
@@ -69,13 +71,16 @@
     "lint-staged": "10.5.1",
     "mongoose": "5.10.9",
     "mysql": "2.18.1",
+    "mysql2": "2.2.5",
```
Comment (BrunnerLivio) [inline package.json:74]: What is this for?

**Comment by BrunnerLivio** [comment]
btw: please don't use past tense in git commits. In general, at Nest, we like to use the Angular commit standards, more info [here](https://github.com/nestjs/terminus/blob/master/CONTRIBUTING.md#-commit-message-guidelines). I'd also like to add [semantic-release](https://github.com/semantic-release/semantic-release) to this library soon, so package releases are more frequent & automated!

File: package.json (hunk 1 above)
Comment (Tony133) [reply package.json:74]: For Sequelize to work as written in the NestJS documentation --> https://docs.nestjs.com/techniques/database#sequelize-integration

**Comment by Tony133** [comment]
You could also use this: https://github.com/conventional-changelog/commitlint

that I oblige you when committing to use sematics in commits, if already on this nestjs package --> https://github.com/nestjs/serve-static

**Comment by BrunnerLivio** [comment]
@Tony133 yeah I think makes sense to add a Pipeline step + precommit hook :)

File: package.json (hunk 1 above)
Comment (BrunnerLivio) [reply package.json:74]: Ah gotcha, nice!

**Comment by BrunnerLivio** [comment]
@Tony133 any clue why we're getting type errors in the Pipeline? :)

**Comment by Tony133** [comment]
Perhaps this is the problem

```TS2416 error: The "model" property in the "Sequelize" type is not assignable to the same property in the "Sequelize" base type.```

But I'm not sure

**Comment by Tony133** [comment]
@BrunnerLivio  I made some changes, i used sequelize version 5 as version 6 gives at the time of problems, now only one pipeline fails do you know how to fix it?

**Comment by Tony133** [comment]
wrong pull

**Pull request closed without merging**